app/Http/Controllers/Consumables/ConsumableCheckoutController.php

```diff
@@ -87,15 +87,15 @@ public function store(Request $request, $consumableId)
 
         // Update the consumable data
         $consumable->assigned_to = e($request->input('assigned_to'));
-
-        $consumable->users()->attach($consumable->id, [
-            'consumable_id' => $consumable->id,
-            'user_id' => $admin_user->id,
-            'assigned_to' => e($request->input('assigned_to')),
-            'note' => $request->input('note'),
-        ]);
-
-        event(new CheckoutableCheckedOut($consumable, $user, Auth::user(), $request->input('note')));
+        for($qty=0;$qty< $request->input('assigned_qty'); $qty++) {
+            $consumable->users()->attach($consumable->id, [
+                'consumable_id' => $consumable->id,
+                'user_id' => $admin_user->id,
+                'assigned_to' => e($request->input('assigned_to')),
+                'note' => $request->input('note'),
+            ]);
+                event(new CheckoutableCheckedOut($consumable, $user, Auth::user(), $request->input('note')));
```

Comment:
Would this trigger an individual email for each item checked out? We probably only want to send one email with the QTY included in the email.

Reply:
This has been fixed, only one email triggered per checkout now.